### PR TITLE
Updating terms and capitalization

### PR DESCRIPTION
**Pull request by rachel-dtr**
## Description

This is the continuation of the effort to update capitalization across the site to be consistent, as well as updating some terms to be correct/less redundant.

### CODE REVIEW

**Comment by dandds** [comment]
Ah, actually this needs a rebase @rachel-dtr . There's a merge conflict in the translations file. I or someone else can help tomorrow if you need it.

**Comment by montana-mil** [comment]
i fixed the conflict :)

File: atst/forms/portfolio_member.py [hunk 1]
```diff
@@ -13,28 +13,28 @@ class PermissionsForm(BaseForm):
     member = StringField()
     user_id = HiddenField()
     perms_app_mgmt = SelectField(
-        None,
+        "App management",
```
Comment (drummond-dds) [inline atst/forms/portfolio_member.py]: Depending on how aggressively we want to use `translations.yml` we can also use them in forms.

See https://github.com/dod-ccpo/atst/blob/master/atst/forms/ccpo_review.py#L13

Comment (montana-mil) [reply atst/forms/portfolio_member.py]: that's a good call

Comment (rachel-dtr) [reply atst/forms/portfolio_member.py]: ok I think I did that...

**Pull request merged**